src/Apps/Auction/routes.tsx

```diff
@@ -22,6 +23,50 @@ export const routes: RouteConfig[] = [
     path: "/auction-faq",
     Component: AuctionFAQ,
   },
+  {
+    path: "auction/:saleID/bid(2)?/:artworkID",
+    Component: ConfirmBidRoute,
+    render: ({ Component, props }) => {
+      if (Component && props) {
```

Comment:
When using `render` default router behavior is overwritten. This stuff should be checked inside of `ConfirmBidRoute` component.

Reply:
Interesting. I think that might be a good optimization to consider later for both this and the Register route cc @dblandin 

Thanks!

Reply:
@damassi Cool! Yeah, that's a great idea.